### ISSUE TITLE
ValueError when loading multiple tf.contrib.keras models in the same scope at different times

### ISSUE THREAD

**Issue by Sergio0694** [tensorflow/tensorflow#20696]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: Yes
- **OS Platform and Distribution**: Linux Ubuntu 16.04
- **TensorFlow installed from**: binary
- **TensorFlow version (use command below)**: 1.8.0
- **Python version**: 3.5.2
- **Bazel version**: N/A
- **CUDA/cuDNN version**: CUDA 7.5, cuDNN 9.0
- **GPU model and memory**: GTX1080 (8GB)
- **Exact command to reproduce**: See [**this snippet**](https://gist.github.com/Sergio0694/aa36c7ed94091ce5503ad908b142aaf0)

### Describe the problem
TensorFlow throws a `ValueError: You are trying to load a weight file containing 16 layers into a model with 0 layers` when trying to create multiple instances of a pretrained Keras model from within the same scope. This only happens if the instances are created one at a time, reopening the same scope multiple times, and works if all the instances are created consecutively, after opening the scope a single time.

Of course, in a real world scenario these instances are created from different places and not all at the same time while building the model, so one would actually need to reopen that scope multiple times.
Hope this helps and that the snippet is clear enough!

### Source code / logs
See [**this snippet**](https://gist.github.com/Sergio0694/aa36c7ed94091ce5503ad908b142aaf0), same one posted under system information.

Here's the stack trace for the snippet above:
```
Traceback (most recent call last):
  File "/home/sergio/Documents/code/keras_repro.py", line 21, in <module>
    vgg2 = load_vgg19(t)
  File "/home/sergio/Documents/code/keras_repro.py", line 12, in load_vgg19
    m = tf.contrib.keras.applications.VGG19(weights='imagenet', include_top=False, input_tensor=x)
  File "/home/sergio/.local/lib/python3.5/site-packages/tensorflow/python/keras/_impl/keras/applications/vgg19.py", line 234, in VGG19
    model.load_weights(weights_path)
  File "/home/sergio/.local/lib/python3.5/site-packages/tensorflow/python/keras/_impl/keras/engine/network.py", line 1190, in load_weights
    saving.load_weights_from_hdf5_group(f, self.layers)
  File "/home/sergio/.local/lib/python3.5/site-packages/tensorflow/python/keras/_impl/keras/engine/saving.py", line 697, in load_weights_from_hdf5_group
    ' layers.')
ValueError: You are trying to load a weight file containing 16 layers into a model with 0 layers.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Bazel version

**Comment by Sergio0694** [comment]
Done 👍
Not sure how to trigger the bot to re-check the message, or if it's just automatic @tensorflowbutler

**Comment by tensorflowbutler** [comment]
Nagging Assignee @jart: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @jart: It has been 33 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @jart: It has been 49 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @jart: It has been 64 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @jart: It has been 79 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @jart: It has been 94 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by emikulic** [comment]
Looks related to https://github.com/tensorflow/tensorflow/issues/20073

**Comment by tensorflowbutler** [comment]
Nagging Assignee @jart: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @jart: It has been 30 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @jart: It has been 45 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by ymodak** [comment]
This is fixed with tf-nightly version '1.15.0-dev20190726'. Thanks!
